.prettierrc.yaml

```diff
@@ -0,0 +1,10 @@
+# See https://prettier.io/docs/en/options. most are set to default
```

Comment:
Config for prettier - per comment: most are set to default. We can change this to JSON TOML or plain js if anyone has an aversion to yaml